src/MasterSite.php

```diff
@@ -530,6 +531,23 @@ function ( $item ) {
 			}
 		);
 
+		// Donate button menu dropdown.
+		// If no Donate menu is defined, we use the old settings from Planet 4 > Donate.
+		$donate_menu_items[] = [
+			'link'  => planet4_get_option( 'donate_button' ) ?? '#',
```

Comment:
I since found out that this function has a [second param for default value](https://github.com/greenpeace/planet4-master-theme/blob/master/functions.php#L73), so here instead of using `??` you could write `planet4_get_option( 'donate_button', '#' )`... Same on the following line 🙂